qt_gui_cpp/CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ find_package(ament_cmake REQUIRED)
 if(WIN32)
   message(STATUS "${PROJECT_NAME} is not yet supported on Windows. Package will not be built.")
   ament_package()
+  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/plugin.xml"
+    "<library path=\"src\"/>")
+  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/plugin.xml"
+    DESTINATION share/${PROJECT_NAME})
```

Comment:
@dirk-thomas mind to add a comment explaining why this change and/or referencing this ticket or the ticket to resolve this lack of support?

Reply:
The `package.xml` file is being installed which references a `plugin.xml` file which wasn't available on Windows. That resulted in the warnings mentioned in the referenced ticket. Until we find a way to completely skip a package on some platforms it needs to be made sure that whatever is installed on such a platform is as consistent as possible.

I don't know what exact comment should be added to the CMake code. If you have something specific in mind please make a suggestion.

Reply:
> I don't know what exact comment should be added to the CMake code. If you have something specific in mind please make a suggestion.

Your first paragraph would've been just fine. Anyways, no biggie.